Dockerfile

```diff
@@ -12,32 +12,33 @@ RUN adduser --home ${CODEQL_HOME} ${USERNAME} && \
     apt-get update && \
     apt-get upgrade -y && \
     apt-get install -y --no-install-recommends \
-    	software-properties-common \
-    	vim \
-    	curl \
-    	wget \
-    	git \
-    	build-essential \
-    	unzip \
-    	apt-transport-https \
-        python3.8 \
-    	python3-venv \
-    	python3-pip \
-    	python3-setuptools \
-        python3-dev \
-    	gnupg \
-    	g++ \
-    	make \
-    	gcc \
-    	apt-utils \
-        rsync \
-    	file \
-        dos2unix \
-    	gettext && \
-        apt-get clean && \
-        rm -f /usr/bin/python /usr/bin/pip && \
-        ln -s /usr/bin/python3.8 /usr/bin/python && \
-        ln -s /usr/bin/pip3 /usr/bin/pip 
+    software-properties-common \
+    nodejs \
+    vim \
+    curl \
+    wget \
+    git \
+    build-essential \
+    unzip \
+    apt-transport-https \
+    python3.8 \
+    python3-venv \
+    python3-pip \
+    python3-setuptools \
+    python3-dev \
+    gnupg \
+    g++ \
+    make \
+    gcc \
+    apt-utils \
+    rsync \
+    file \
+    dos2unix \
+    gettext && \
+    apt-get clean && \
+    rm -f /usr/bin/python /usr/bin/pip && \
+    ln -s /usr/bin/python3.8 /usr/bin/python && \
```

Comment:
Not sure about the pip3 symlinking, but can this line not be replaced by also installing the `python-is-python3` package?

Reply:
That's beyond the scope of this PR. All I want is TypeScript support and to fix the broken build. I'll leave that up to the maintainers

Reply:
I'll incorporate this comment into my next PR, thanks @intrigus-lgtm!